code/game/gamemodes/clock_cult/clock_structures/ocular_warden.dm

```diff
@@ -134,3 +134,14 @@
 	target = null
 	visible_message("<span class='warning'>[src] settles and seems almost disappointed.</span>")
 	return 1
+
+/obj/structure/destructible/clockwork/ocular_warden/get_efficiency_mod()
```

Comment:
this proc has an argument for if the returned value is for multiplication or division, and it needs to be properly supported.